### PR TITLE
Pset 1. Hello world.

### PR DESCRIPTION
**Pull request by AlexOrd**

### CODE REVIEW

**Comment by petr-like** [comment]
test

File: pset1/hello.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('Hello world!');
```
Comment (petr-like) [inline pset1/hello.js:1]: test

**Comment by ivan-krot** [comment]
Test_comment

**Review by petr-like — COMMENTED** [review]
test1

**Comment by VikaAnt** [comment]
Test

**Pull request merged**